azkaban-common/src/test/java/azkaban/executor/InteractiveTestJob.java

```diff
@@ -24,7 +26,7 @@
 
 public class InteractiveTestJob extends AbstractProcessJob {
 
-  private static final ConcurrentHashMap<String, InteractiveTestJob> testJobs =
+  public static final ConcurrentHashMap<String, InteractiveTestJob> testJobs =
```

Comment:
Here's the PR for reverting public to private: https://github.com/azkaban/azkaban/pull/1163